test/test_ast_generation.py

```diff
@@ -22,6 +22,8 @@
     ('assignment_false', 'a = False'),
     ('assignment_none', 'a = None'),
     ('assignment_true', 'a = True'),
+    ('assignment_paren', '(a) = 42'),
```

Comment:
I would add also some correct examples with annotations:

```
(x) : int
(x): int = 34
```

Reply:
But there are such examples already. Should I add more?

Reply:
Oh, I may have missed then. If they already exists, ignore what I said